tests/std/tests/P0528R3_cmpxchg_pad/test.cpp

```diff
@@ -92,14 +162,19 @@ struct X20 {
     void set(char v) {
         x = v;
         std::memset(&y, 0, sizeof(y));
-        z = 0;
+        z = ~v;
+    }
+
+    bool check(const char v) const {
+        return x == v && z == ~v;
     }
 };
 
 
 template <class X, std::size_t S>
 void test() {
     static_assert(sizeof(X) == S, "Unexpected size");
+    static_assert(!std::has_unique_object_representations_v<X>, "No padding type");

```

Comment:
This test should directly include `<type_traits>` for `std::has_unique_object_representations_v`.